tiledb/sm/query/ast/query_ast.cc

```diff
@@ -251,8 +294,9 @@ Status ASTNodeVal::check_node_validity(const ArraySchema& array_schema) const {
   }
 
   // Ensure that for a set membership test has members that all match the
```

Comment:
I can't quite parse this comment. Is it supposed to be "Ensure that a set membership test..."?

Reply:
I updated this to hopefully be more clear. The gist is basically just that if we're apply a set against a fixed size field, I wanted to make sure that every set member was the correct width.